nuklear.h

```diff
@@ -5726,15 +5726,6 @@ static nk_handle fictional_handle = {0};
 #define NK_ASSERT(expr) assert(expr)
 #endif
 
-#ifndef NK_MEMSET
-#define NK_MEMSET nk_memset
-#endif
-#ifndef NK_MEMCPY
-#define NK_MEMCPY nk_memcopy
-#endif
-#ifndef NK_SQRT
-#define NK_SQRT nk_sqrt
-#endif
```

Comment:
Shouldn't we extend this to all other `nk_` function which are separately `#ifdef`ed? I think those are `nk_sin()` `nk_cos()` `nk_strtod()` and `nk_dtoa()`.

Reply:
Can add those, too - if the way I did fix the others is what would be accepted.

Reply:
>if the way I did fix the others is what would be accepted.

I think it's OK as in the documentation they're not scattered, but on one place. So we don't need to have them at one place in the source.